### DISCUSSION TITLE
id.4 USA says not able to authorize using credentials

### DISCUSSION THREAD

**Discussion by dharmareddyrakesh** [tillsteinbach/VWsFriend#272] [Q&A]
Hello,
I am using this to connect to my 2021 Id.4 USA model. 
It saying my credentials are invalid. Can you confirm if the USA models supported in this?

**Comment by dharmareddyrakesh** [comment]
My Logs-
INFO:session_manager:Could not use token from file /tmp/weconnect.token ([Errno 2] No such file or directory: '/tmp/weconnect.token')
docker-vwsfriend-1   | 2022-07-07T17:24:35+0000:CRITICAL:vwsfriend_base:There was a problem when authenticating with WeConnect: Error during login, account myemailid@ismasked.com does not exist
docker-vwsfriend-1   | 2022-07-07T17:25:36+0000:INFO:vwsfriend_base:vwsfriend 0.19.2 (using WeConnect-python 0.44.2)

**Reply by tillsteinbach** [reply]
Yes, unfortunately VW is using completely different servers and I have no access to an US car out of Europe to find out.

**Reply by dharmareddyrakesh** [reply]
Any thing I can help to figure this out for you?

**Reply by tillsteinbach** [reply]
Would you be willing to share your account for myVW for some days? That would be maybe a chance to see if the US cars are compatible enough.

**Reply by dharmareddyrakesh** [reply]
I can add you as a additional driver.  Please dm me on private.

**Reply by tillsteinbach** [reply]
I just had a look in the internals of the myVW App again and I think it does not make sense to try it with your account for now. It seems like you have a completely different backend in the states. So the API is 80% different than what we have in Europe.
I hope that will change and I can make it compatible at a later point in time.

**Reply by dharmareddyrakesh** [reply]
Can you please share details on where I can find API? and how to access it! 
I can write up my own console app to consume data

**Reply by tillsteinbach** [reply]
Can I contact you somewhere? Then I can give you some details.

**Reply by dharmareddyrakesh** [reply]
Add me on discord, so we can talk! My username is drr#5459.

**Reply by tillsteinbach** [reply]
ah I'm not on discord, a mail address would be fine.

**Reply by dharmareddyrakesh** [reply]
Proemail915@gmail.com
